src/blueprint.ts

```diff
@@ -238,6 +239,27 @@ export class Blueprint {
     this.props = Blueprint.responseToBlueprintProps(response);
   }
 
+  /**
+   * Chooses the preferred Zk Framework according to the provided example email
+   * sets props.zkFramework
+   * @param exampleEml - The exampleEml for this blueprint
+   */
+  async assignPreferredZkFramework(exampleEml: string) {
```

Comment:
Simce we are using eml everywhere, it would be better to use eml here as well insttead of `exampleEml`